src/Command/Core/CoreConfigureCommand.php

```diff
@@ -254,16 +186,14 @@ private function configureAdapter(InputInterface $input, OutputInterface $output
      */
     private function isCredentialsValid($name, array $adapter, array $config)
     {
-        $application = $this->getApplication();
         /** @var \Gush\Application $application */
-        $application->setConfig($this->config);
-
+        $application = $this->getApplication();
         $factory = $application->getAdapterFactory();
 
         if ($adapter[AdapterFactory::SUPPORT_REPOSITORY_MANAGER]) {
-            $adapter = $factory->createRepositoryManager($name, $config, $this->config);
+            $adapter = $factory->createRepositoryManager($name, $config, $this->getApplication()->getConfig());
```

Comment:
or in the minimum we could use $application->getConfig() instead of $this->getApplication()->getConfig();